Cargo.lock

```diff
@@ -3328,6 +3328,7 @@ dependencies = [
  "itertools",
  "jobserver",
  "libc",
+ "lock_api",
```

Comment:
We can use `parking_lot::lock_api::*` instead

Reply:
Done.